src/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard App</title>
+    <!-- <link rel="stylesheet" type="text/css" href="./assets/style.css" /> -->
```

Comment:
- [ ] Following the best practices, you are not meant to leave unused code as a comment in your codebase. I'll suggest you remove this as it makes your codebase look a bit unprofessional. :+1: :small_airplane: